crates/ruff_python_formatter/src/lib.rs

```diff
@@ -29,11 +30,12 @@ mod options;
 pub(crate) mod other;
 pub(crate) mod pattern;
 mod prelude;
+mod shared_traits;
 pub(crate) mod statement;
 pub(crate) mod type_param;
 mod verbatim;
 
-include!("../../ruff_formatter/shared_traits.rs");
+// include!("../../ruff_formatter/shared_traits.rs");
```

Comment:
I'd suggest removing this line entirely.

Reply:
Ah yeah derp. That was an accident.